### DISCUSSION TITLE
10xGenomice_Arc_Multiomes Issue_addGeneExpressionMatrix to proj with my seRNA file, get no overlap

### DISCUSSION THREAD

**Discussion by harrisSe6** [GreenleafLab/ArchR#1102] [Questions / Documentation]
I am getting no overlap with my seRNA and atac data. No sure why and new to ArchR. Here is my log files as of today. Thanks for any suggestion.steve

[ArchR-createArrows-58b42b584473-Date-2021-10-24_Time-13-20-22.log](https://github.com/GreenleafLab/ArchR/files/7406358/ArchR-createArrows-58b42b584473-Date-2021-10-24_Time-13-20-22.log)
[ArchR-addGeneExpressionMatrix-58b449c94281-Date-2021-10-24_Time-16-05-51.log](https://github.com/GreenleafLab/ArchR/files/7406359/ArchR-addGeneExpressionMatrix-58b449c94281-Date-2021-10-24_Time-16-05-51.log)

**Comment by rcorces** [comment]
Thanks for using ArchR but I'm afraid I do not have enough information to help you. Please make sure the tutorial works for you and that you have read the full manual.

Most likely cause is that your gene names in your seRNA dont match the gene names in your geneAnnotation

**Comment by harrisSe6** [comment]
Thanks Ryan,  will check gene names in seRNA and my mm10 annotation file.  Also, my run yesterday was with 2 libraries –two different states combine with Aggr in cellranger. Will try a run with one library and check  gene name issue. steve

**Reply by rcorces** [reply]
ArchR is not designed to work with CellRanger Aggr

**Comment by harrisSe6** [comment]
Thanks Ryan, very helpful. Checked my seRNA and geneAnnoMm10 and seem ok. Here is a copy.  So, I guess I should  run  the individual samples-libraries  separately for now.  Will give it a try. steve

**Comment by harrisSe6** [comment]
Hi Ryan, making some progress with ArchR, use 1  10x Multiomice sample—PSC_B7d(stem cells differentiated to day with rBmp2). Got the seRNA into proj and got the Doublet  working, removed 3.6% of cells. Got the LSI-ATAC to run, but having issues with the LSI-RNA—maybe something to do with the “GeneExpressionMatrix”.  Not sure, cp the last few lines of the output.  Let me know if you have any suggestion to try. Almost read to start work. OC looks good, etc  Thanks for any help and let me know if you need any more info. Here is log file also.  Working with the 10X Multiomic PBMC and full manual. steve
  Working on making a cellColData file also.

**Comment by harrisSe6** [comment]
Hi Ryan, made significant progress with one sample.  3 Questions,  1st how do you add columns to the seRNA data, currently has 0 column names. Looked at getCellColData, and see have GEX_nUMI so some is in there. Attached.  Got thru to ch.7, and stuck at ch8—getting the scATAC and scRNA integrated.

2nd Question, I have two samples and will try to analyze in ArchR. These two samples are from a time course experiment, and want to analyze together and look at differences at the two time points. How do I do that.
3rd Question, in the ArchRBrowser, limit is 250kb, and many of my enhancers in the Bmp2 gene are 1 million basepairs, in this big gene desert. Maybe you can modify.  I did go out to 1 million in the ch. 7.6 and worked fine. Also, the plot of the peaks in the bmp2 gene TAD seem strange, ie with all the “peaks” in for example  cluster 1—red. Can you change the scale so I can get rid of the noise and also, maybe not autoscale each cluster is my guess is what it might be doing. Thanks for the help. See what I have done so far in .pdf.
steve

**Comment by harrisSe6** [comment]
Hi Ryan, still having the same issue with making a LSI_RNA with the one sample. Any suggestions? Stev

projPSC_B7d_3e <- addIterativeLSI(ArchRProj=projPSC_B7d_3d,clusterParams=list(resolution=0.7, sampleCells=3747,n.start=10), useMatrix="GeneExpressionMatrix", depthCol="Gex_nUMI", varFeatures=2500, firstSelection= "variable", binarize=F, name="LSI_RNA")
Checking Inputs...
ArchR logging to : ArchRLogs\ArchR-addIterativeLSI-4086ae81000-Date-2021-10-31_Time-14-08-43.log
If there is an issue, please report to github with logFile!
2021-10-31 14:08:45 : Computing Variability Across All Features, 0.003 mins elapsed.
2021-10-31 14:08:46 : Computing Variable Features, 0.022 mins elapsed.
###########
2021-10-31 14:08:47 : Running LSI (1 of 2) on Top Features, 0.033 mins elapsed.
###########
2021-10-31 14:08:47 : Creating Partial Matrix, 0.033 mins elapsed.

************************************************************
2021-10-31 14:08:51 : ERROR Found in .LSIPartialMatrix for
LogFile = ArchRLogs\ArchR-addIterativeLSI-4086ae81000-Date-2021-10-31_Time-14-08-43.log

<simpleError in .logThis(cellNames, "cellNames supplied", logFile = logFile): object 'logFile' not found>

************************************************************

Error in .logError(e, fn = ".LSIPartialMatrix", info = "", errorList = errorList,  :
  Exiting See Error Above
>
